src/components/ChatEntry.js

```diff
@@ -1,22 +1,35 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp.js';
 
 const ChatEntry = (props) => {
+  const buttonText =
+    props.liked ? '🏳️‍⚧️' : '🤍';
+  
+  let entryClass = 'chat-entry local'  
+  
+  if (props.sender !== 'Lichen') {
+    entryClass = 'chat-entry remote';
+  }
+
```

Comment:
Nice! We need this in order to determine which sender's messages go to the right or left